package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "crud",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "parcel ./src/index.html",
+    "build": "rm -rf dist; parcel build --detailed-report",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fmt": "prettier --write .",
+    "fix": "prettier --write src/; eslint --fix src/",
+    "lint": "prettier --check ."
+  },
+  "source": "src/index.html",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ParkingReformNetwork/crud.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
You need to use the Apache 2 license. We already have the LICENSE file in the repo.